tfexec/cmd.go

```diff
@@ -63,10 +67,12 @@ func envSlice(environ map[string]string) []string {
 }
 
 func (tf *Terraform) buildEnv(mergeEnv map[string]string) []string {
+	osEnv := envMap(os.Environ())
```

Comment:
you should restore this to the old code, since its not always necessary, and this invokes a syscall even if thrown away and some mutex's, etc.

Reply:
Thanks, this was left over from an unused change